### PR TITLE
Import command

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
After this, #64 will need just one more PR to have some commands add built-in imports.

### CODE REVIEW

File: Source/Languages/Casing/CaseStyle.ts [hunk 1]
```diff
@@ -11,14 +11,29 @@ namespace GLS.Languages.Casing {
...
+
+        /**
+         * Packages in lower case, such as "my.variable".
+         */
+        PackageLowerCase,
+
+        /**
+         * Packages in upper case, such as "My.Variable".
+         */
+        PackageUpperCase,
```
Comment (faulda) [inline Source/Languages/Casing/CaseStyle.ts]: Should we call this something unrelated to code? 
Like "PeriodSeparatedUpperCase" or something like that. There could be other things that use this format besides packages.

File: Source/Languages/Casing/CaseStyle.ts [hunk 2]
```diff
@@ -11,14 +11,29 @@ namespace GLS.Languages.Casing {
...
-         * Camel case, such as myVariable.
+         * File system case, such as "my/variable.
          */
-        CamelCase,
+        FileSystem,
+
+        /**
+         * Packages in lower case, such as "my.variable".
+         */
+        PackageLowerCase,
```
Comment (faulda) [inline Source/Languages/Casing/CaseStyle.ts]: Same comment as for PackageUppercase.

File: Source/Languages/Casing/CaseStyle.ts (hunk 1 above)
Comment (faulda) [inline Source/Languages/Casing/CaseStyle.ts]: Something like "PackageCase" seems like a language property that would be of type CaseStyle.

**Comment by faulda** [comment]
General note on the conversions - it seems like most of the cases we're converting follow a common pattern - each one does something like 
`firstword wordseparator `*`[nextword wordseparator, ...]`*` `*`lastword`*
We could structure the base converter to follow this format. At that point, each CaseStyle simply becomes a set of properties:

- how to style the first word
- how to style the middle/next words
- how to style the last word
- what the word separator character (if any) is

Under this style, for example:
PackageUpperCase = style all words upper case, word separator = `.`
camelCase = style first word lower case, next and last words upper case, word separator none
etc.

File: Source/Languages/Casing/CaseStyle.ts (hunk 1 above)
Comment (JoshuaKGoldberg) [inline Source/Languages/Casing/CaseStyle.ts]: Sure, let's finalize #70 then revisit this.
